spice/ext4mag.py

```diff
@@ -0,0 +1,145 @@
+# ext4mag.py
```

Comment:
is that different from the other extmag scripts?

Reply:
Yes, this one was created to illustrate use through Python to run Magic.